File: server/src/main/java/org/cleanas2/cmd/CommandBase.java
package org.cleanas2.cmd;

import org.cleanas2.cmd.CommandResult;

/**
 * @author Andrew Backer {@literal awbacker@gmail.com / andrew.backer@powere2e.com}
 */
public abstract class CommandBase {
    public abstract CommandResult run(String... params) throws Exception;
}

File: server/src/main/java/org/cleanas2/cmd/CommandBase.kt
package org.cleanas2.cmd

import org.cleanas2.cmd.CommandResult

/**
 * @author Andrew Backer awbacker@gmail.com / andrew.backer@powere2e.com
 */
abstract class CommandBase {
    @Throws(Exception::class)
    abstract fun run(vararg params: String): CommandResult
}
